src/Services/OrderService.php

```diff
@@ -760,6 +761,49 @@ public function complete($order)
         );
     }
 
+    /**
+     * Remove the bundle prefix from the order item name.
+     * Default prefix is "[BUNDLE] "
+     * 
+     * @param string $name
+     * @return string
+     */
+    public function removeBundlePrefix(string $name): string
+    {
+        return $this->removeOrderItemPrefix($name, \Plenty\Modules\Order\Models\OrderItemType::TYPE_ITEM_BUNDLE);
+    }
+
+    /**
+     * Remove the bundle component prefix from the order item name.
+     * Default prefix is "[-] "
+     * 
+     * @param string $name
+     * @return string
+     */
+    public function removeBundleComponentPrefix(string $name): string
+    {
+        return $this->removeOrderItemPrefix($name, \Plenty\Modules\Order\Models\OrderItemType::TYPE_BUNDLE_COMPONENT);
+    }
+
+    /**
+     * @param string $name
+     * @param int $orderItemTypeId
+     * @return string
+     */
+    private function removeOrderItemPrefix(string $name, int $orderItemTypeId): string
+    {
+        /** @var OrderSettingsRepositoryContract $settingsRepository */
+        $settingsRepository = pluginApp(OrderSettingsRepositoryContract::class);
+        $settings = $settingsRepository->get();
+        $prefix = $settings->orderItemPrefixes[$orderItemTypeId] ?? null;
+
+        if ($prefix !== null && (substr($name, 0, strlen($prefix)) == $prefix)) {
+            $name = substr($name, strlen($prefix));
```

Comment:
Was ist mit evtl. vorhandenem Leerzeichen " " zwischen dem Wort und der Bezeichnung?
In der Regel sieht die Artikelbezeichnung ja so aus: `"[-] Name of the variation"`. Sollte es nicht mit entfernt werden?

Reply:
evtl. so:
```suggestion
            $name = trim(substr($name, strlen($prefix)));
```

Reply:
Es wäre dann halt die Frage, ob wir davon ausgehen, dass das Leerzeichen mit in dem Präfix steht, oder nicht. Aktuell steht mit drin.